pkg/lifecycle/startup/startup.go

```diff
@@ -170,9 +173,15 @@ func Run() {
 				utils.Terminate()
 			}
 		}
+
+		k8sNode, err = clientset.CoreV1().Nodes().Get(ctx, k8sNodeName, metav1.GetOptions{})
+		if err != nil {
+			log.WithError(err).Error("Failed to read Node from datastore")
+			utils.Terminate()
+		}
 	}
 
-	configureAndCheckIPAddressSubnets(ctx, cli, node)
+	configureAndCheckIPAddressSubnets(ctx, cli, node, k8sNode, DEFAULT_INTERFACES_TO_EXCLUDE)
```

Comment:
I think we should revert this change to pass DEFAULT_INTERFACES_TO_EXCLUDE all the way through the call stack of the autodetection functions. It seems to be an extra argument just to pass a constant value.